PaymentHighway/UI/ExpirationDateTextField.swift

```diff
@@ -11,6 +11,11 @@ open class ExpiryDateTextField: TextField {
 
     private var cardExpiryDateDeleting = false
 
+    lazy var expiryDatePicker: ExpiryDatePickerView = {
```

Comment:
Could this be `private(set) lazy var...`?

Reply:
Good! I set it as private since is used only within the class.